wolfssl/ssl.h

```diff
@@ -1155,6 +1155,9 @@ WOLFSSL_API void wolfSSL_SetIOWriteFlags(WOLFSSL* ssl, int flags);
     WOLFSSL_API void wolfSSL_SetIO_NetX(WOLFSSL* ssl, NX_TCP_SOCKET* nxsocket,
                                       ULONG waitoption);
 #endif
+#ifdef HAVE_LWIP_NATIVE
+    #include "lwip/tcp.h"
```

Comment:
Why is this header include needed at ssl.h level?  Couldn't it be lower, like io.c or internal.[ch].